mergeable/src/GitHub.ts

```diff
@@ -0,0 +1,21 @@
+const OPEN_PRS = { state: 'open' }
+
+const fetchOpenPullRequests = (gitHubApi: GitHubApi.Instance) => async (githubSecrets: GitHubSecrets): Promise<any> => {
```

Comment:
nope I didn't type that layer of the `github-api` module, I probably should 👍  and then maybe submit a PR to `typings`